pkg/onboarding/usecases/profile.go

```diff
@@ -169,6 +169,8 @@ type ProfileUseCase interface {
 	SwitchUserFlaggedFeatures(ctx context.Context, phoneNumber string) (*dto.OKResp, error)
 
 	FindUserByPhone(ctx context.Context, phoneNumber string) (*profileutils.UserProfile, error)
```

Comment:
There is already `FindUserByPhone` query

Reply:
Yes, this is `findUsers`